gcs/get_gcs.go

```diff
@@ -21,6 +21,12 @@ type Getter struct {
 	// Timeout sets a deadline which all GCS operations should
 	// complete within. Zero value means no timeout.
 	Timeout time.Duration
+
+	// FileSizeLimit limits the size of an single
+	// decompressed file.
+	//
+	// The zero value means no limit.
+	FileSizeLimit int64
```

Comment:
This was probably left-over from a previous patch I shared. This shouldn't be required.
```suggestion
```

Reply:
Thanks for calling that out. I saw it was not being used so was hoping the review would shed some light on it. I'll remove.